src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-types/lancie-admin-ticket-types.html

```diff
@@ -0,0 +1,245 @@
+<link rel="import" href="../../../../bower_components/paper-listbox/paper-listbox.html">
+<link rel="import" href="../../../../bower_components/paper-item/paper-item.html">
+
+<link rel="import" href="../../lancie-admin-page-layout.html">
+<link rel="import" href="../../../lancie-admin-table/lancie-admin-table.html">
+
+<dom-module id="lancie-admin-ticket-types">
+  <template>
+    <style include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        display: block;
+      }
+
+      paper-card {
+        width: calc(100% - 32px);
+        margin: 16px;
+      }
+
+      [hidden] {
+        display: none !important;
+      }
+
+      @media (max-width: 640px) {
+        paper-card {
+          width: 100%;
+          margin: 0;
+        }
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="getTypeAjax" refurl="tickets/types" on-lancie-ajax="onTypeResponse">
+    </lancie-ajax>
+
+    <lancie-ajax id="ajaxAddOptionToType" method="POST" body="[[getOptionBody(selectedOption)]]" refurl="tickets/types/[[selectedId]]/option"></lancie-ajax>
+
+    <lancie-ajax id="ajaxChangeBuyableOfType" method="POST" body="[[getBuyableBody(buyableBool)]]" refurl="tickets/types/[[selectedId]]/buyable"></lancie-ajax>
+
+    <paper-card heading="Tickets/types">
+      <div class="card-content">
+        <p>Here is where the types will be.</p>
+        <p>Requests will be on <code>/api/v1/tickets/types</code></p>
+
+        <lancie-admin-table id="table" column-mappings="[[mappings]]" column-renderers="[[renderers]]"></lancie-admin-table>
+      </div>
+      <div class="card-actions">
+        <lancie-error id="noTypeSelectedError"></lancie-error>
+        <paper-button on-tap="openAddTypeDialog">Add Ticket Type</paper-button>
+        <paper-button on-tap="openAddOptionToTypeDialog">Add Ticket Option To Type(s)</paper-button>
+        <paper-button on-tap="openChangeBuyableDialog">Change Buyable</paper-button>
+      </div>
+    </paper-card>
+
+    <lancie-dialog id="addTicketTypeDialog">
+      <h2>Add Ticket Type</h2>
+      <div class="dialog-content">
+        <lancie-error id="addTypeDialogError"></lancie-error>
+        <lancie-form id="addTicketTypeForm" refurl="tickets/types" on-response="onAddTypeResponse">
+          <paper-input label="Name" name="name" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Text" name="text" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Price" name="price" step=".01" type="number" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Number Available" name="numberAvailable" type="number" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Deadline" name="deadline" type="datetime-local" auto-validate required error-message="This field is required!"></paper-input>
```

Comment:
Just to confirm, setting a time here results in a ticket with the correct time being added?

Reply:
Yes